events/lastAvatarStandingTrivia/entityServerScripts/triviaMasterServer.js

```diff
@@ -405,15 +407,16 @@
         });
     };
 
-    this.plusHFC = function(){      
+    this.plusHFC = function(){     
+        console.log(JSON.stringify("PLUS HFC.................", coin.position)); 
         Entities.addEntity({
             type: "ParticleEffect",
             name: "Trivia Pot Increase Particle",            
             lifetime: 4,
             collidesWith: "",
             collisionMask: 0,
             collisionless: true,
-            position:   Vec3.sum(coin.position, {x: -1, y: 3, z: -2}),
+            position: {x:-6.4341,y: -8.8567, z: 61.5412},// Vec3.sum(coin.position, Vec3.multiplyQbyV(coin.rotation, {x: 1, y: 0, z: 2})),
```

Comment:
remove comment

Reply:
This is still needed for trivia in the trivia domain, it was only removed for multi-con